src/ggsn_gn.erl

```diff
@@ -119,7 +119,7 @@ handle_event({call, From}, terminate_context, _State, Data) ->
 
 handle_event({call, From}, {path_restart, Path}, _State,
 	     #{context := #context{path = Path}} = Data) ->
-    close_pdp_context(normal, Data),
+    close_pdp_context(path_restart, Data),
```

Comment:
`path_restart` is a bad name for an externally visible the termination cause. This really is a `peer restart`